src/ol/uri.js

```diff
@@ -0,0 +1,17 @@
+goog.provide('ol.uri');
+
+
+/**
+ * Appends query parameters from a map to an URI.
+ *
+ * @param {string} uri The original URI, which may already have query data.
```

Comment:
From what I can tell the original `goog.uri.utils.appendParamsFromMap` only does append no merge.